application/btsieve/src/settings/mod.rs

```diff
@@ -87,4 +95,15 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn can_deserialize_log_level() -> Result<(), failure::Error> {
+        let settings = Settings::create("./config/default.toml");
```

Comment:
📣 Please update this config file with the new configuration.

Also, it'd be good to update the regtest config file accordingly.

Reply:
> 📣 Please update this config file with the new configuration.

but it is already updated? Or what do you mean?

but yes, can add it to the regtest configs as well.

Reply:
> > 📣 Please update this config file with the new configuration.
> 
> but it is already updated? Or what do you mean?
> 
> but yes, can add it to the regtest configs as well.

My apologies, looks like I did not see it!